### PR TITLE
fix: fixed issues of skeletonLoader component

### PR DESCRIPTION
**Pull request by adnan1naeem**
skeletonLoader fixed

### CODE REVIEW

File: src/components/SkeletonLoader/index.native.tsx [hunk 1]
```diff
@@ -17,19 +17,19 @@ export interface SkeletonProps {
...
-						x={`${x ? x : ''}`}
-						y={`${y ? y : ''}`}
-						rx={`${x ? rx : ''}`}
-						ry={`${ry ? ry : ''}`}
-						width={`${width ? width : ''}`}
-						height={`${height ? height : ''}`}
-					/>
-				)}
+				<Svg.Rect
+					x={`${x}`}
```
Comment (artalat) [inline src/components/SkeletonLoader/index.native.tsx:25]: why not `x={x}`?

Comment (adnan1naeem) [reply src/components/SkeletonLoader/index.native.tsx:25]: This has been fixed in code

Comment (artalat) [reply src/components/SkeletonLoader/index.native.tsx:25]: where?

**Pull request merged**